lib/templates/android_screen_base.tt

```diff
@@ -16,37 +16,37 @@ class AndroidScreenBase < Calabash::ABase
-      # screen element method using the element name which is the
-      # method name without the first 'touch_' chars
-      touch_screen_element public_send(method.to_s.sub('touch_', ''))
-    elsif method.to_s.start_with?('enter_')
-      # If method starts with enter_, execute the enter method using
-      # the field name, which is the method name without the initial
-      # 'enter_' chars and appended '_field' chars
+  def <%= I18n.translate( "screen_base.method_missing" ) %> (method, *args)
+    if method.to_s.<%= I18n.translate "screen_base.start_with?" %>('<%= I18n.translate "screen_base.touch_" %>')
+      # <%= I18n.translate( "comments.method_missing_1" ) %>
+      # <%= I18n.translate( "comments.method_missing_2" ) %>
+      # <%= I18n.translate( "comments.method_missing_3" ) %>
+      <%= I18n.translate( "screen_base.touch_screen_element" ) %> public_send(method.to_s.sub('<%= I18n.translate"screen_base.touch_" %>', ''))
+    elsif method.to_s.<%= I18n.translate "screen_base.start_with?" %>('enter_')
+      # <%= I18n.translate( "comments.starts_enter_1" ) %>
+      # <%= I18n.translate( "comments.starts_enter_2" ) %>
+      # <%= I18n.translate( "comments.starts_enter_3" ) %>
       enter args[0], public_send("#{method.to_s.sub('enter_', '')}_field")
-    elsif method.to_s.end_with?('_visible?')
-      # If method ends with _visible?, executes the visible? method
-      # The field name is the method name without de ending
-      # '_visible? chars
-      visible? public_send(method.to_s.sub('_visible?', ''))
-    elsif method.to_s.end_with?('_visible!')
-      # Do the same as the method above, but throws an exception
-      # if the field is not visible
-      field_name = method.to_s.sub('_visible!', '')
+    elsif method.to_s.<%= I18n.translate "screen_base.end_with?" %>('<%= I18n.translate "screen_base._visible?" %>')
+      # <%= I18n.translate( "comments.termina_com_visible_1" ) %>
+      # <%= I18n.translate( "comments.termina_com_visible_2" ) %>
+      # <%= I18n.translate( "comments.termina_com_visible_3" ) %>
+      <%= I18n.translate( "screen_base.visible?" ) %> public_send(method.to_s.sub('<%= I18n.translate "screen_base._visible?" %>', ''))
+    elsif method.to_s.<%= I18n.translate "screen_base.end_with?" %>('<%= I18n.translate "screen_base._visible!" %>')
+      # <%= I18n.translate( "comments.termina_com_visible_4" ) %>
+      # <%= I18n.translate( "comments.termina_com_visible_5" ) %>
+      <%= I18n.translate( "screen_base.field_name" ) %> = method.to_s.sub('<%= I18n.translate "screen_base._visible!" %>', '')
                          .sub('_field', '')
                          .sub('_', ' ')
                          .capitalize
       raise ElementNotFoundError, "ID: #{field_name}" unless
-        visible? public_send(method.to_s.sub('_visible!', ''))
+        <%= I18n.translate( "screen_base._visible?" ) %> public_send(method.to_s.sub('<%= I18n.translate "screen_base._visible!" %>', ''))
```

Comment:
I would not add the `?` and `!` in the translations. This will avoid problems if anyone forgot to add they in the translations, `visible?` and `visible!` have different behaviours. I think that would be better this way:

```
<%= I18n.translate( "screen_base._visible" ) %>? public_send(method.to_s.sub('<%= I18n.translate "screen_base._visible" %>!', ''))
```

What do you think?